chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -397,6 +397,31 @@ public void leaveConversation(@NonNull final Conversation conversation,
     }
 
     /**
+     * Delete a conversation.
+     *
+     * @param conversation  the conversation
+     * @param callback     the callback
+     */
+    public void deleteConversation(@NonNull final Conversation conversation,
+                                  @Nullable final DeleteCallback<Conversation> callback) {
+        this.skygear.callLambdaFunction("chat:delete_conversation",
+                new Object[]{conversation.getId()},
+                new LambdaResponseHandler() {
+                    @Override
+                    public void onLambdaSuccess(JSONObject result) {
+                        callback.onSucc(conversation);
```

Comment:
In JS SDK, the promise just return true. But here is returning the deleted conversation.

Since we are trying to keep consistence across SDK, we should also return true here.